src/index.html

```diff
@@ -20,8 +20,355 @@
+              </div>
+              <div class="card__reviews">Reviews: 5</div>
+            </section>
+            <section class="card__price">
+              <div class="card__price-text">Price:</div>
+              <div class="card__price-price">$2,199</div>
+            </section>
+            <a
+              href="#"
+              class="card__link"
+              data-qa="card-hover"
+            >
+              <button class="card__button">BUY</button>
+            </a>
+          </div>
+        </section>
+        <section
+          class="card"
+          data-qa="card"
+        >
+          <section class="card__image"></section>
+          <div class="card__wrapper">
+            <section class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </section>
+            <section class="card__code">Product code: 195434</section>
+            <section class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews">Reviews: 5</div>
+            </section>
+            <section class="card__price">
+              <div class="card__price-text">Price:</div>
+              <div class="card__price-price">$2,199</div>
+            </section>
+            <a
+              href="#"
+              class="card__link"
+              data-qa="card-hover"
+            >
+              <button class="card__button">BUY</button>
+            </a>
+          </div>
+        </section>
+        <section
+          class="card"
+          data-qa="card"
+        >
+          <section class="card__image"></section>
+          <div class="card__wrapper">
+            <section class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </section>
+            <section class="card__code">Product code: 195434</section>
+            <section class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__reviews">Reviews: 5</div>
+            </section>
+            <section class="card__price">
+              <div class="card__price-text">Price:</div>
+              <div class="card__price-price">$2,199</div>
+            </section>
+            <a
+              href="#"
+              class="card__link"
+              data-qa="card-hover"
+            >
+              <button class="card__button">BUY</button>
+            </a>
+          </div>
+        </section>
```

Comment:
Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_catalog/assets/106397016/79fd72bf-725a-475a-ac02-7df818e69f56)

Reply:
I've done what I've could with this moment, but "npm test" ruin these moments because of lint-testing